README.md

```diff
@@ -52,6 +53,14 @@ to https://localhost:8000/. Credentials for the 'admin' user can be found in
 `./deploy/k8s/central-deploy/password`
 (`deploy/openshift/central-deploy/password` in the OpenShift case).
 
+### Helm Charts
+
+Alternatively, there are also opensource flavor [Helm Charts](https://helm.sh/) available at [github.com/stackrox/helm-charts](https://github.com/stackrox/helm-charts/tree/main/opensource) which can be used to install StackRox
```

Comment:
I appreciate you starting this, however, I believe, this should be done differently in order to bring real benefit for community users.
I described the idea [here](https://github.com/stackrox/stackrox/pull/1864#issuecomment-1141003041):
> Remember, our goal is to make entry barrier as low as possible for community users. This README can follow a typical pattern of open source projects such as first introduce the product (you don't have to write that), next explain how to install it (your part), next explain how to develop for it (currently existing content).

If you read the `## Deployment` section above, it will become clear that it assumes the installing user has to clone the source code of this repo.
a) This is already quite an entry barrier to require _just any user_ to install git and clone the repo in order to install the product.
b) This deploys just the current revision and not necessarily the stable release.

I'd ask you to try impersonate a stranger user who wants to spend as less time as possible to get the product installed and working for them. This means we need to move Helm installation to be the first thing, and, actually, the only one in the context of `## Quickstart installation`.

Secondly, instructions for secured cluster services are missing. Please add that too, otherwise the product isn't deployed completely. You may learn how to do that by following the official instructions yourself, see https://docs.openshift.com/acs/installing/installing_helm/install-helm-quick.html

Next, these instructions will likely not work normally until there are official release charts published to the repo. Therefore, merging this PR now will lead to more confusion than actually help users. Let's think how we can publish opensource charts for 3.70.0 to that location (another JIRA task to be created). Otherwise, we need to wait until everything happens automatically as part of 3.71.0, in 9 weeks from now.

Finally, I'd ask you to provide a link at the end of this section to the official online product docs for users who want to learn more about customizing Helm installation: https://docs.openshift.com/acs/installing/installing_helm/install-helm-customization.html

Reply:
Fair enough, I'll rework this.